src/components/Link/README.md

```diff
@@ -23,54 +23,35 @@ keywords:
 
 # Link
 
-Links are used to embed actions or pathways to more information in a sentence.
+Links take users to another place, and usually appear within or directly following a sentence.
 
----
-
-## Best practices
-
-Links should:
-
-- Consist of text that clearly describes either the action that merchants will take or the location they’ll navigate to.
-- Only be used in a sentence. For stand-alone navigational actions, [use the button component](https://polaris.shopify.com/components/actions/button).
+For actions that aren't related to navigation, use the [button component](/components/actions/button).
 
 ---
 
-## Content guidelines
-
-The link component should follow the general [content guidelines](https://polaris.shopify.com/content/actionable-language#section-links) for links.
-
-### Links
-
-Links should:
-
-- Set the expectation of where merchants will be taken or what action will be performed.
-- Be consistent with the content they point to. If a navigational link leads to a page called Orders, label the link “orders”.
-- Use descriptive text and avoid using generic content such as “click here”.
-
-<!-- usageblock -->
+## Best practices
 
-#### Do
+### Buttons versus links
 
-Learn more about <a>fulfilling orders</a> at the Shopify Help Center
+Links are used primarily for navigation, and usually appear within or directly following a sentence.
 
-#### Don’t
+Buttons are used primarily for actions, such as “Add”, “Close”, “Cancel”, or “Save”. Plain buttons, which look similar to links, are used for less important or less commonly used actions, such as "view shipping settings".
 
-<a>Click here</a> to learn more about fulfilling orders
+The HTML for buttons and links carries specific meaning. Using these components intentionally and consistently results in:
```

Comment:
Making the first part of this sentence clearly refer to React components could clarify the intent:

```suggestion
The markup resulting from `Button` and `Link` carries meaning. Using these components intentionally and consistently results in:
```

What do you think?